### PR TITLE
Updated requirements for python 3.8 support

### PR DESCRIPTION
**Pull request by emanuelschuetze**
updated:
channels to 2.3.1
channels-redis to 2.4.1
daphne to 2.4.0

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein Can you please check the Changelogs of channels, channels-redis and daphne:

https://github.com/django/channels/blob/master/CHANGELOG.txt
https://github.com/django/channels_redis/blob/master/CHANGELOG.txt
https://github.com/django/daphne/blob/master/CHANGELOG.txt

My quick tests with python3.8 + redis was successfully.

**Comment by normanjaeckel** [comment]
Maybe you have to add Python 3.8 tests to .travis.yml (and maybe also to dependency installing at the top of this file).

**Comment by normanjaeckel** [comment]
See also line 15 in travis.yml.

**Comment by tsiegleauq** [comment]
What I tested so far:
- revert changes in production.txt and big_mode.txt
-> All tests pass
- Only update daphne
-> Error:  daphne 2.4.0 has requirement asgiref~=3.2, but you'll have asgiref 2.3.2 which is incompatible.
- Set daphne to 2.4
-> ERROR: daphne 2.3.0 has requirement asgiref~=3.0, but you'll have asgiref 2.3.2 which is incompatible
- Add asgiref>=3.0
-> ERROR: channels 2.1.7 has requirement asgiref~=2.3, but you'll have asgiref 3.2.3 which is incompatible.
-> ERROR: channels-redis 2.3.3 has requirement asgiref~=2.1, but you'll have asgiref 3.2.3 which is incompatible.
- Remove asgiref>=3.0, set channels to 2.3
-> ERROR: channels-redis 2.3.3 has requirement asgiref~=2.1, but you'll have asgiref 3.2.3 which is incompatible.
- set channels-redis>=2.2,<2.5
-> pip success
-> Tests fail

using:

- channels 2.2.0
- daphne 2.3.0
- channels-redis: 2.4.1

Apparently, there are already breaking changes in the next minor versions.

**Comment by tsiegleauq** [comment]
I found out that using a file-based database during testing fixes the "table is locked" error. See the commit I made.
Not sure if Travis is fine with file based databases tho.

**Comment by tsiegleauq** [comment]
We are passing Travis again :tada: 
@normanjaeckel or @FinnStutzenstein please check if my changes are okay.
I also cleaned up the files that have not previously been `black`ed.

File: requirements/big_mode.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Requirements for Redis and PostgreSQL support
-channels-redis>=2.2,<2.4
+channels-redis>=2.2,<2.5
```
Comment (ostcar) [inline requirements/big_mode.txt]: channels-redis uses pools now:
https://github.com/django/channels_redis/blob/8efee75564ef36dd5388cfbcaff5bc5b12e76a0b/CHANGELOG.txt#L35

I don't think there will be problems with our redis pool. But maybe we could refactor our pool to use the channels pool:
https://github.com/OpenSlides/OpenSlides/blob/bb41125bd146af1e367fa0d9eeeb3adf49646cea/openslides/utils/redis.py#L36:L54

Comment (FinnStutzenstein) [reply requirements/big_mode.txt]: We are currently using these pools: https://github.com/OpenSlides/OpenSlides/blob/bb41125bd146af1e367fa0d9eeeb3adf49646cea/openslides/utils/redis_connection_pool.py#L5

**Pull request merged**